internal/image/resource_image.go

```diff
@@ -728,3 +921,38 @@ func splitImageResourceID(id string) (string, string) {
 	pieces := strings.SplitN(id, ":", 2)
 	return pieces[0], pieces[1]
 }
+
+func exactlyOne(in ...bool) bool {
+	var count int
+	for _, b := range in {
+		if b {
+			count++
+		}
+	}
+	return count == 1
+}
+
+func packImageDir(path string) (string, error) {
```

Comment:
I wouldn't add support for that, basically nobody ever used the directory image packing thing of the Incus CLI.

We should just support existing image files.

Reply:
Removed.